### PR TITLE
[lib32-mesa-git] fixing the build

### PR DESCRIPTION
**Pull request by SolarAquarion**

### CODE REVIEW

**Comment by yuyichao** [comment]
Could you please figure out how rebase works……..

File: archlinuxcn/lib32-mesa-git/lilac.yaml [hunk 1]
```diff
@@ -1,15 +1,18 @@
+---
```
Comment (yuyichao) [inline archlinuxcn/lib32-mesa-git/lilac.yaml:1]: Is this needed?

**Comment by SolarAquarion** [comment]
No, but the yaml linter wanted it

File: archlinuxcn/lib32-mesa-git/lilac.yaml (hunk 1 above)
Comment (SolarAquarion) [reply archlinuxcn/lib32-mesa-git/lilac.yaml:1]: the linter wanted that

**Comment by yuyichao** [comment]
Ok, sound good enough to me. If lilac have problems using it I’ll let @lilydjwg complain about it…

**Pull request merged**